qcodes/instrument/sims/AMI430.yaml

```diff
@@ -0,0 +1,106 @@
+spec: "1.0"
```

Comment:
How do we generate a simulated instrument from this yaml file? Maybe this file: https://pyvisa-sim.readthedocs.io/en/latest/ is helpful for people.